setup.cfg

```
@@ -3,7 +3,7 @@ universal = 1
 
 [flake8]
 exclude = docs
-ignore = E203,E501,W293,W503
+ignore = W503,E302,E203,F841
```

E203 wasn't included in our "ignore" [list](https://paper.dropbox.com/doc/2020-10-05-DSL-Refactor--BA4JkdZnAZDKey2Q5MnOIkBAAg-wi9QHuxpkXeeVk8JATCcX).

```suggestion
ignore = W503,E302,F841
```